deploy/sc.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Do have a look at my earlier comments on yamls.
Should we merge related yamls into a single yaml file.

Reply:
done.